Framework/DataHandling/src/LoadMuonNexus2.cpp

```diff
@@ -234,6 +234,11 @@ void LoadMuonNexus2::doExec() {
       double dum = detector.getFloat("first_good_time");
       setProperty("FirstGoodData", dum);
     }
+
+    if (detector.containsDataSet("last_good_time")) {
+      double dum = detector.getFloat("last_good_time");
```

Comment:
this is not in the nxs2 files either. So lets just set it to the largest x value